scripts/cnode-helper-scripts/env

```diff
@@ -89,6 +85,8 @@ OFFLINE_MODE='N'
 [[ $(basename $0 2>/dev/null) = "cnode.sh" ]] && OFFLINE_MODE='Y' # for backwards compatibility
 [[ $(basename $0 2>/dev/null) = "topologyUpdater.sh" ]] && OFFLINE_MODE='Y' # for backwards compatibility
 
+[[ -z ${BRANCH} ]] && { [[ -f "${PARENT}"/.env_branch ]] && BRANCH=$(cat "${PARENT}"/.env_branch) || BRANCH=master; }
```

Comment:
Should we inherit BRANCH from parent environment variable (just got caught out coz I sourced env file before in my session using alpha branch)?

Reply:
What about -b flag on runtime?
For let's say topologyUpdater.